token/core/identity/msp/wm.go

```diff
@@ -0,0 +1,220 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package msp
+
+import (
+	view2 "github.com/hyperledger-labs/fabric-smart-client/platform/view"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/flogging"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/identity"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/identity/msp/common"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/identity/msp/idemix"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/identity/msp/x509"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/driver"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/driver/config"
+	"github.com/pkg/errors"
+)
+
+var logger = flogging.MustGetLogger("token-sdk.driver.identity.tms")
+
+const (
+	// OwnerMSPID is the default MSP ID for the owner wallet
+	OwnerMSPID = "OwnerMSPID"
+	// IssuerMSPID is the default MSP ID for the issuer wallet
+	IssuerMSPID = "IssuerMSPID"
+	// AuditorMSPID is the default MSP ID for the auditor wallet
+	AuditorMSPID = "AuditorMSPID"
+	// CertifierMSPID is the default MSP ID for the certifier wallet
+	CertifierMSPID = "CertifierMSPID"
+)
+
+// IdentityType is the type of identity
+type IdentityType int
+
+const (
+	// LongTermIdentity is the type of the long term identity (x509)
+	LongTermIdentity IdentityType = iota
+	// AnonymousIdentity is the type of the anonymous identity (Idemix
```

Comment:
mention that this is also a long term identity

Reply:
okay